packages/amplify-e2e-core/src/categories/lambda-function.ts

```diff
@@ -472,3 +474,12 @@ const getRuntimeDisplayName = (runtime: FunctionRuntimes) => {
       throw new Error(`Invalid runtime value: ${runtime}`);
   }
 };
+
+export function validateRemovalNodeModulesDir(projRoot) {
```

Comment:
nit: `validateNodeModulesDirRemoval` is a little easier to read

Reply:
done!!